uint32  id             # id сообщения 
uint32[4] micric       # Данные по уонцевикам